### ISSUE TITLE
dateRange gets translated to incorrect value

### ISSUE THREAD

**Issue by lauraalkhoury-youitv** [cube-js/cube.js#348]
(using v0.13.12)

In my query, I pass in the following `timeDimension` filter:
```
{
  dimension: "<some dimension>",
  dateRange: "from 23 hours ago to now"
}
```

But instead, the data returned is from midnight to now, and the `timeDimension` returned as part of the resultSet's query field has been changed to:
```
{
  dimension: "<some dimension>"
  dateRange: [
    "2020-01-15T00:00:00.000"
    "2020-01-15T23:59:59.999"
  ]
}
```

Note that this happens regardless of the initial value in the `timeDimension` filter, ie. it also happens if "from 7 days ago to now" is selected.

Here are some screenshots of an example:
![Screen Shot 2020-01-14 at 4 24 12 PM](https://user-images.githubusercontent.com/56270203/72447195-e413cb00-3782-11ea-80dd-ebdb6988cdc9.png)
![Screen Shot 2020-01-14 at 4 24 28 PM](https://user-images.githubusercontent.com/56270203/72447204-e7a75200-3782-11ea-8e39-237674a3f5f9.png)

**Comment by lauraalkhoury-youitv** [comment]
Upon further investigation I noticed Cube also transforms "from 7 days ago to now" to [2020-01-15T00:00:00Z, 2020-01-15T23:59:59Z], ie. today only.

I reverted back to v0.10.12 and it seems to be getting the last week of data correctly, but the issue i outlined in the description above still remains in both versions.

Logs:
[cube-logs-1day.txt](https://github.com/cube-js/cube.js/files/4065762/cube-logs-1day.txt)
[cube-logs-1wk.txt](https://github.com/cube-js/cube.js/files/4065763/cube-logs-1wk.txt)
